addons/web/static/src/core/datetime/datetimepicker_service.js

```diff
@@ -428,6 +428,10 @@ export const datetimePickerService = {
                 /** @type {DateTimePickerProps} */
                 const rawPickerProps = {
                     ...DateTimePicker.defaultProps,
+                    onReset: () => {
+                        updateValue(pickerProps.value.length === 2 ? [false, false] : false);
```

Comment:
Wouldn't it crash if `pickerProps.value` is a date time object instead of an array ?